src/bundle/Resources/views/link_manager/view.html.twig

```diff
@@ -50,19 +50,19 @@
                     <td class="ez-table__cell">{{ 'url.label.last_checked'|trans }}</td>
                     <td class="ez-table__cell">
                         {% if url.lastChecked %}
-                            {{ url.lastChecked|localizeddate('medium', 'short', null, ez_user_settings['timezone']) }}
```

Comment:
I'd say `localizeddate('medium', 'short', ...)` is corresponding to `full` format right now. We are using `full` format in almost all places. `short` was used in the Timeline and in UDW only.

Reply:
`localizeddate('medium', 'short'` == `ez_short_datetime`
`localizeddate('medium', 'medium'` == `ez_long_datetime`

Reply:
Currently 'medium', 'short' is like this:
![screenshot at mar 05 13-21-32](https://user-images.githubusercontent.com/12594013/53805179-bbf3f280-3f49-11e9-8a5d-2b22921a7ee3.png)

so `localizeddate('medium', 'short'` is more like full not short.